vignettes/Vaccines.Rmd

```diff
@@ -40,153 +123,262 @@ simparams <- get_parameters(list(
 
 simparams <- set_equilibrium(simparams, starting_EIR)
 
+# Run a model with no interventions in a setting with no seasonality
```

Comment:
I think in the `get_parameters` model above, the arguments need to be `clinical_incidence_rendering_min_ages` to go with the output you want to plot?